var/spack/repos/builtin/packages/openssh/package.py

```diff
@@ -61,3 +63,33 @@ def configure_args(self):
         # install step and fail if they cannot do so.
         args = ['--with-privsep-path={0}'.format(self.prefix.var.empty)]
         return args
+
+    def install(self, spec, prefix):
+        make('install', parallel=False)
+
+    def setup_build_environment(self, env):
```

Comment:
I wonder if we can make this a bit nicer in spack... a separate setup_test_environment wouldn't hurt.

Can you for now do a quick return `if not self.run_tests` ?

Reply:
or define you own setup_test_environment and call that conditionally from setup_build_environment if self.run_tests

Reply:
Likewise, very good idea - openssh: setup_test_environment() is done!